### PR TITLE
refs #120: add templates for GitLab issues.

### PR DESCRIPTION
**Pull request by noemi-mancini**

### CODE REVIEW

**Comment by stickgrinder** [comment]
@noemi-mancini is this PR missing something or can it be merged?

**Comment by noemi-mancini** [comment]
@stickgrinder I just added some refinements. Now it should be ready to merge.

File: content/tools-and-policies/gitlab-issue-templates.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+- [Issue templates](#introduction)
+- [Feature](#feature)
+- [Bug](#bug)
+- [Techical task](#technical-task)
+- [Support](#support)
+
+## Introduction
+When you create a new project on GitLab, even before creating the first issue, we strongly advise you to add these templates, in order to immediately have issues structured in the correct way.  
+Templates must be added to the `.gitlab/issue_templates` folder of the project root.
```
Comment (stickgrinder) [inline content/tools-and-policies/gitlab-issue-templates.md:16]: It's better to explain that those templates should be saved as separate files.
A nice addition would be to upload a zip with the three files to unpack in the repo.

**Comment by stickgrinder** [comment]
LGTM! Merged

**Pull request merged**